src/gwt/src/org/rstudio/studio/client/workbench/views/output/lint/LintManager.java

```diff
@@ -314,9 +315,6 @@ public void onError(ServerError error)
    
    private void showLint(LintContext context, JsArray<LintItem> lint)
    {
-      if (docDisplay_.isPopupVisible() || !docDisplay_.isFocused())
```

Comment:
I don't recall exactly why this code was here but it'd be worth confirming that this doesn't interfere with e.g. autocompletion popup menus (e.g. if you were to start typing something, press <TAB> to request completions and the popup menu is shown -- what happens if lint gets shown at the same time?)

Reply:
Good catch, didn't mean to cart blanch remove that whole line. We do want to run lint on background documents, however, so I've left the focus param out.